File: goLang/operators.go
package main

import "fmt"

func operators() {
	a, b := 10, 3

	// Arithmetic
	fmt.Println("Addition: ", a+b)
	fmt.Println("Deduction: ", a-b)
	fmt.Println("Multiplication: ", a*b)
	fmt.Println("Division: ", a/b)
	fmt.Println("Modulus: ", a%b)

	// Comparison
	fmt.Println("Equal: ", a == b)
	fmt.Println("Not Equal: ", a != b)
	fmt.Println("Less Than: ", a > b)
	fmt.Println("Greater Than: ", a < b)
	fmt.Println("Less Than or Equal: ", a <= b)
	fmt.Println("Greater Than or Equal: ", a >= b)

	// Unary
	a++
	fmt.Println("Only Post Increment: ", a)
	b--
	fmt.Println("Only Post Increment: ", b)

	x, y := true, false
	// Logical Operator
	fmt.Println("Logical And: ", x && y)
	fmt.Println("Logical OR: ", x || y)
	fmt.Println("Logical Not: ", !x)

	// Bitwise Operator
	fmt.Println("Bitwise And: ", a&b)
	fmt.Println("Bitwise Or: ", a|b)
	fmt.Println("Bitwise XOr: ", a^b)
	fmt.Println("Bitwise Left Shift: ", a>>1)
	fmt.Println("Bitwise Right Shift: ", b<<1)

	// Assignment
	a += 5
	b *= 3
	fmt.Println("Assignments: ", a, b)
}

File: Java/Operators.java
import static java.lang.System.out;

public class Operators {
    public static void main(String[] args) {
        int a = 6, b = 9;

        // Arithmetic Operators
        out.println("Addition: " + (a + b));
        out.println("Deduction: " + (a - b));
        out.println("Multiplication: " + (a * b));
        out.println("Division: " + (a / b));
        out.println("Modulus: " + (a % b));

        // Comparison
        out.println("Equal: " + (a == b));
        out.println("Not Equal: " + (a != b));
        out.println("Greater Than: " + (a > b));
        out.println("Less Than: " + (a < b));
        out.println("Greater Than Equal: " + (a >= b));
        out.println("Less Than Equal: " + (a <= b));

        // Logical Operator
        boolean x = true, y = false;
        out.println("Logical And: " + (x && y));
        out.println("Logical OR: " + (x || y));
        out.println("Logical Not: " + (!x));

        // Bitwise
        out.println("Bitwise And: " + (a & b));
        out.println("Bitwise Or: " + (a | b));
        out.println("Bitwise XOr: " + (a ^ b));
        out.println("Bitwise Not: " + (~b));
        out.println("Bitwise Left Shift: " + (a << 1));
        out.println("Bitwise Right Shift: " + (a >> 1));

        // Assignment
        a += 5;
        b *= 4;
        out.println("Assignment a: " + a);
        out.println("Assignment b: " + b);
    }
}